### ISSUE TITLE
Error executing luv callback: attempt to concatenate field 'payload_data' (a nil value)

### ISSUE THREAD

**Issue by mcepl** [glacambre/firenvim#1265]
- OS Version: linux x86-64
- Browser Version:  Firefox 94.0
- Browser Addon Version: 0.2.11
- Neovim Plugin Version: 0.2.11

### What I tried to do
Just using Firenvim.

### What happened
Periodically this error message happens:

![error-message](https://user-images.githubusercontent.com/198999/145428530-c564855f-f9cc-4786-b22c-81e7d1f64bb3.png)

**Comment by glacambre** [comment]
Thanks for the report, this is the same issue as #1253: the browser sends a websocket "ping" to which neovim has to reply with "pong". I had never implemented this part of the protocol until recently and now I have, but with what looks like a bug.

This usually happens when you don't interact with a Firenvim frame for some time (not sure how long though).

**Comment by mcepl** [comment]
> This usually happens when you don't interact with a Firenvim frame for some time (not sure how long though).

Yes, quite often when I write a comment on some ticket (or on Reddit when not in the work mode), I need to collect some information elsewhere, so I may be in a textbox for the extensive period of time.

Or perhaps not, I got the error message now even before I wrote the previous paragraph and that I wrote just straight away without much thinking or researching.

**Comment by glacambre** [comment]
This is fixed by #1267 :).

**Comment by mcepl** [comment]
I wonder whether the cure is not worse than a sickness here. It seems that instead of random messages I get periodically all text lost (without storing it in undo-list, so it is completely lost).

**Comment by glacambre** [comment]
Right, I just noticed this while answering another issue. I've reverted c8cd51c3e04a1b1a12cfa21b6b9aaf62c16b983f in 804d6c4bbbdc7f4b679617f88f8136e3b38bee37 until I figure out what's going wrong.

Thanks for the report!

**Comment by abeforgit** [comment]
Hi, I'm having this issue as well, and I'm a bit surprised to not see more people here with the same issue. 
I wonder if people found a workaround, or a way to silence the error, or maybe it went away for some range of neovim versions?

I'm also on nixos, so it's highly possible there is something wrong with my configuration. Is this still a known issue?

**Comment by glacambre** [comment]
Haven't seen it in a very long while. What neovim version+firenvim browser addon+firenvim plugin version are you using?
